specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2022-02-14/imagebuilder.json

```diff
@@ -0,0 +1,1563 @@
+      "x-ms-discriminator-value": "Shell",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ImageTemplateCustomizer"
+        }
+      ],
+      "properties": {
+        "scriptUri": {
+          "type": "string",
+          "description": "URI of the shell script to be run for customizing. It can be a github link, SAS URI for Azure Storage, etc"
+        },
+        "sha256Checksum": {
+          "type": "string",
+          "default": "",
+          "description": "SHA256 checksum of the shell script provided in the scriptUri field"
+        },
+        "inline": {
+          "type": "array",
+          "description": "Array of shell commands to execute",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ImageTemplateRestartCustomizer": {
+      "type": "object",
+      "description": "Reboots a VM and waits for it to come back online (Windows). Corresponds to Packer windows-restart provisioner",
+      "x-ms-discriminator-value": "WindowsRestart",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ImageTemplateCustomizer"
+        }
+      ],
+      "properties": {
+        "restartCommand": {
+          "type": "string",
+          "description": "Command to execute the restart [Default: 'shutdown /r /f /t 0 /c \"packer restart\"']"
+        },
+        "restartCheckCommand": {
+          "type": "string",
+          "description": "Command to check if restart succeeded [Default: '']"
+        },
+        "restartTimeout": {
+          "type": "string",
+          "description": "Restart timeout specified as a string of magnitude and unit, e.g. '5m' (5 minutes) or '2h' (2 hours) [Default: '5m']"
+        }
+      }
+    },
+    "ImageTemplateWindowsUpdateCustomizer": {
+      "type": "object",
+      "description": "Installs Windows Updates. Corresponds to Packer Windows Update Provisioner (https://github.com/rgl/packer-provisioner-windows-update)",
+      "x-ms-discriminator-value": "WindowsUpdate",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ImageTemplateCustomizer"
+        }
+      ],
+      "properties": {
+        "searchCriteria": {
+          "type": "string",
+          "description": "Criteria to search updates. Omit or specify empty string to use the default (search all). Refer to above link for examples and detailed description of this field."
+        },
+        "filters": {
+          "type": "array",
+          "description": "Array of filters to select updates to apply. Omit or specify empty array to use the default (no filter). Refer to above link for examples and detailed description of this field.",
+          "items": {
+            "type": "string"
+          }
+        },
+        "updateLimit": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
>int32 [](http://example.com/codeflow?start=21&length=5)

waiting for breaking changes approval before signing off. Rest of the ARM changes look fine.

Reply:
This was removed by request of breaking change reviewer, however it is triggering a failure in Swagger LintDiff tests. Awaiting breaking change reviewer to approve this removal.